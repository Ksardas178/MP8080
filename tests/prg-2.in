LXI H o200 o000
MVI B o007
MOV A B
SUB M
MOV B A
MOV M A
HLT