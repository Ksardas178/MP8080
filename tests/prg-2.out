
[32;1mInput is correct
[0m
[30;1mCode analysis results:
[0m______________________________
00000000: LXI 4 128 0
00000001: MVI 0 7
00000010: MOV 7 0
00000011: SUB 6
00000100: MOV 0 7
00000101: MOV 6 7
00000110: HLT
